### PR TITLE
CHE-1002 Add target UX improvements

### PR DESCRIPTION
**Pull request by vitaliy-guliy**
**_1 Upvote**_ Signed-off-by: Vitaliy Guliy vguliy@codenvy.com

### CODE REVIEW

File: plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/targets/TargetsPresenter.java [hunk 1]
```diff
@@ -700,24 +740,32 @@ protected void onErrorReceived(Throwable exception) {
      * Ensures machine is started.
      */
     private void onConnected(final String machineId) {
-        machineService.getMachine(machineId).then(new Operation<MachineDto>() {
+        // There is a little bug in machine service on the server side.
+        // The machine info is updated with a little delay after running a machine.
+        // Using timer must fix the problem.
+        new Timer() {
```
Comment (vparfonov) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/targets/TargetsPresenter.java:746]: Not clear about timer, how it solve problem? Why you set 500 not 600? Can accept it as urgent fix but must be fixed ASAP. Other ok for me

Comment (garagatyi) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/targets/TargetsPresenter.java:746]: If you say that there is a bug in machine API than create an issue and assign it to me, please.

Comment (vitaliy-guliy) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/targets/TargetsPresenter.java:746]: I'm not sure where is a problem. But when I receive a RUNNING state via websocket I ask the machine service about the machine and get CREATING state. Only waiting for a while gives the result.

Comment (vitaliy-guliy) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/targets/TargetsPresenter.java:746]: Sure it's a temporary desigion and has to be fixed ASAP.

**Comment by ashumilova** [comment]
+1

**Pull request merged**